syntax = "proto3";

package mnist_service;

service MnistService {
    rpc GetTrainingSamples(DataRequest) returns (stream Sample) {}
}
message DataRequest {
    int64 sample_size = 1;
}
message Sample {
    bytes image = 1;
    int64 label = 2;
}
